Runtime/Utilities/CameraCache.cs

```diff
@@ -21,6 +21,16 @@ public static Camera Main
         {
             get
             {
+                if (cachedCamera != null)
```

Comment:
I really want to get rid of this `CameraCache` thing. But for now I am fine with keeping it and fixing it if you see an issue here. I feel like the overall use/behaviour of this script is unclear and it is also a backdoor to the camera we don't want people to use. They should simply use the camera system / camera rig.

Reply:
It feels better not, but I agree.  We should plan in the future to just remove this as Unity fixed the original problem this was meant to solve (I believe)